internal/workflow_testsuite.go

```diff
@@ -543,6 +544,63 @@ func (e *TestWorkflowEnvironment) OnUpsertMemo(attributes interface{}) *MockCall
 	return e.wrapWorkflowCall(call)
 }
 
+func (e *TestWorkflowEnvironment) OnNexusOperation(
+	service string,
+	operation string,
```

Comment:
Arguably this should accept a func the same way mocking an activity does and when a func is present instead of a string you should not have to have pre-registered it.